packages/common/driver-definitions/src/driverError.ts

```diff
@@ -90,11 +90,18 @@ export enum DriverErrorType {
      * "fluidInvalidSchema"
      */
     fluidInvalidSchema = "fluidInvalidSchema",
+
     /**
      * Error indicating an API is being used improperly resulting in an invalid operation.
      * ! Should match the value of ContainerErrorType.usageError
     */
     usageError = "usageError",
+
+    /**
+     * File is locked for read/write by storage, e.g. whole collection is locked and access is denied,
+     * or file is locked for editing
```

Comment:
I'd remove "or file is locked for editing."
Mostly because this scenario does not exist today, but if we add support for it, most likely we will not use this error flow, but rather allow opening file in read-only mode (i.e. open will not fail).